src/test/java/uk/gov/ch/oauth/OAuthCoordinatorTest.java

```diff
@@ -31,6 +32,8 @@
 @ExtendWith(MockitoExtension.class)
 class OAuthCoordinatorTest {
 
+    static final String STATE = "state";
```

Comment:
Remove this as we already have a constant for state

Reply:
changed